serviceregistration/consul/consul_service_registration_test.go

```diff
@@ -492,6 +492,43 @@ func TestConsul_serviceTags(t *testing.T) {
 	}
 }
 
+func TestConsul_serviceMeta(t *testing.T) {
+	tests := []struct {
+		conf map[string]string
+		pass bool
+	}{
+		{
+			conf: map[string]string{},
+			pass: true,
+		},
+		{
+			conf: map[string]string{"service_meta": "true"},
+			pass: false,
+		},
+		{
+			conf: map[string]string{"service_meta": "[{\"key\":\"value\"}]"},
```

Comment:
How come the config must be a list? Since we're only taking the first of these values, I'd expect to be able to configure this as a set of JSON strings, e.g.
```
conf: map[string]string{"service_meta": "{\"key\":\"value\", \"bar\":\"baz\"}"},
```